src/components/piece/piece.css

```diff
@@ -0,0 +1,6 @@
+.piece {
```

Comment:
This is from another jira task and should not be a part of this PR. This is a good example of why we need to get one jira task finished and merged in and then work on another task. If you want to work on another task while waiting for a code review. Create a different branch for that jira task and go from there. Just know that there may be merge conflicts to resolve down the line, but can be a necessary evil :-)